report.php

```diff
@@ -5,8 +5,69 @@
  */
 
 require __DIR__ . '/functions.php';
+require __DIR__ . '/vendor/autoload.php';
 
 // Check required environment variables
 check_required_env();
 
-// @todo process phpunit test run results; send to WordPress.org
+$WPT_SSH_CONNECT = getenv( 'WPT_SSH_CONNECT' );
+$WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
+$WPT_PREPARE_DIR = getenv( 'WPT_PREPARE_DIR' );
+$WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' );
+$WPT_REPORT_API_KEY = getenv( 'WPT_REPORT_API_KEY' );
+
+log_message('Getting SVN Revision');
+$rev = exec('git -C ' . escapeshellarg( $WPT_PREPARE_DIR ) . ' log -1 --pretty=%B | grep "git-svn-id:" | cut -d " " -f 2 | cut -d "@" -f 2');
```

Comment:
perform_operations doesn't appear to return the output, so we can't get the revision from git.